time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -499,7 +496,11 @@ def update(self, id, data: dict, description=None):
         time_entry = self.repository.find(id, event_ctx)
         self.check_whether_current_user_owns_item(time_entry)
```

Comment:
Here we have a usage of this method using `self.` as prefix which AFAIK it means it is going to use the method which is part of the class instance, right?

If so, why should this method be static, I think we can get rid of the `@staticmethod` annotation or  am I wrong?

```
    @staticmethod
    def check_whether_current_user_owns_item(data):
        if data.owner_id is not None and data.owner_id != current_user_id():
            raise CustomError(
                HTTPStatus.FORBIDDEN,
                "The current user is not the owner of this time entry",
            )
```

Reply:
I agree, in the first push the test fell, so I left it like that, now I tried again and the test no longer fell, so I will leave it as suggested.